substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/MethodEntry.java

```diff
@@ -153,6 +179,14 @@ public void updateRangeInfo(DebugInfoBase debugInfoBase, DebugMethodInfo debugMe
         }
     }
 
+    public boolean getIsVirtual() {
```

Comment:
I'd rename this to `isVirtual`.
```suggestion
    public boolean isVirtual() {
```

Reply:
Done.